services/data/src/links/RestAPILink/queryToRequestOptions/xWwwFormUrlencodedMatchers.test.ts

```diff
@@ -0,0 +1,25 @@
+import { isSvgConversion } from './xWwwFormUrlencodedMatchers'
+
+describe('isSvgConversion', () => {
+    it('returns true for a POST to "svg.png"', () => {
+        expect(
+            isSvgConversion('create', {
+                resource: 'svg.png',
+            })
+        ).toEqual(true)
```

Comment:
For comparing primitives I think toBe is what you want: https://jestjs.io/docs/expect#tobevalue. According to the docs toEqual is for deep equality checks on objects.